src/index.ts

```diff
@@ -622,4 +628,4 @@ const themeFile = (options: PostcssThemeOptions = {}) => (
   }
 };
 
-export default postcss.plugin('postcss-themed', themeFile);
+export default themeFile;
```

Comment:
<img width="970" alt="image" src="https://user-images.githubusercontent.com/5761061/97637468-f13f3500-19f7-11eb-8550-8fa92908de52.png">


Looking at the migration guide, I think there might be a couple of recommended attributes we should include like `postCSSPlugin` and `Once`.